app/controllers/user_languages_controller.rb

```diff
@@ -1,19 +1,53 @@
-# class UserLanguagesController < ApplicationController
-#   def options
-#     @user_languages = UserLanguage.find()
-#   end
+class UserLanguagesController < ApplicationController
+  def new_language #new
+    @user_language = UserLanguage.new
+    @all_languages = Language.all
+    capitalize(@all_languages)
+  end
 
-#   # def update_options
-#   #   @user_language =
-#   # end
-#   # code after merging column w Esh
+  #creats languages fine but errors do not render properly
+  def create_language #create
+    @name = params[:user_language][:language].reject(&:blank?) # @name is an array of languages user inputs
+    if @name.blank?
+      flash[:error] = "You must select at least one language"
+      render :new_language
+    else
+      @name.each do |language|
+        @language = Language.where(name: language.downcase)[0]
+        @user_language = UserLanguage.new(user_language_params)
+        @user_language.user = current_user
+        @user_language.language = @language
+        if @user_language.save
+          redirect_to root_path
+        else
+          render :new_language
+        end
+      end
+    end
+  end
 
-#   private
```

Comment:
routing issue keeps looking for 'get'
sessions/queue view page has route to this method added but not tested